MUI/build.gradle

```diff
@@ -149,10 +149,11 @@ build.dependsOn uninstallPreviousMUI
 
 
 task installZip(type: Exec) {
+    dependsOn buildExtension
     group = "Installation"
     description = "Installs the zip package in installed Ghidra's default extension directory"
-    commandLine 'unzip', '-o', "$buildExtension.destinationDir/$buildExtension.baseName.$buildExtension.extension", '-d', ghidraUserDirectory+'/Extensions/'
-    dependsOn buildExtension
+    def zipPath = buildExtension.destinationDirectory.get().getAsFile().toString() + "/" + buildExtension.archiveBaseName.get() + "." + buildExtension.archiveExtension.get()
```

Comment:
gradle 8.0 deprecation warnings aren't shown when building on the default `warning-mode`, but I figured it's something good to change anyways